src/3d/qgsoffscreen3dengine.cpp

```diff
@@ -36,29 +36,8 @@
 #include <Qt3DRender/QViewport>
 #include <QtGui/QOpenGLContext>
 
-
 QgsOffscreen3DEngine::QgsOffscreen3DEngine()
 {
-  // Set up the default OpenGL surface format.
-  QSurfaceFormat format;
-
-  // by default we get just some older version of OpenGL from the system,
-  // but for 3D lines we use "primitive restart" functionality supported in OpenGL >= 3.1
-  // Qt3DWindow uses this - requesting OpenGL 4.3 - so let's request the same version.
-#ifdef QT_OPENGL_ES_2
-  format.setRenderableType( QSurfaceFormat::OpenGLES );
-#else
-  if ( QOpenGLContext::openGLModuleType() == QOpenGLContext::LibGL )
-  {
-    format.setVersion( 4, 3 );
-    format.setProfile( QSurfaceFormat::CoreProfile );
-  }
-#endif
-
-  format.setMajorVersion( 3 );
-  format.setDepthBufferSize( 32 ); // TODO: or 24?  (used by QWindow3D)
-  format.setSamples( 8 );
-  QSurfaceFormat::setDefaultFormat( format );
```

Comment:
Why is this removed - was it causing any problems?

Reply:
I thought it might be better to just have default surface format stuff, I don't see the need to change it

Reply:
There is a comment which explains the reasoning:
```
  // by default we get just some older version of OpenGL from the system,
  // but for 3D lines we use "primitive restart" functionality supported in OpenGL >= 3.1
  // Qt3DWindow uses this - requesting OpenGL 4.3 - so let's request the same version.
```
Can you please check what is the OpenGL version you are getting and what is the default format you are getting?

Reply:
After I turned back this bit, more tests were passing so I think I should just keep it as.